muspinsim/constants.py

```diff
@@ -17,8 +17,8 @@
 EFG_2_MHZ = (
     cnst.physical_constants["atomic unit of electric field " "gradient"][0]
     * cnst.e
-    * 1e-37
-    / cnst.h
+    * 1e-36  # 1e-6 * 10e-31, the latter for converting Q to m^2
+    / cnst.hbar
```

Comment:
Yes this was the source of problems with quadrupole moment calculations. It should be hbar, and the constant value is 1e-6 for converting to MHz and 10e-31 which is a combination of 10e-28 for converting from barn, and due to the quadrupolar moment units it also requires a factor of 1e-3. Its the factor of 10 instead of 1 here that accounts for the difference, and probably why this mistake was made in the first place. I should make the comment clearer though.